app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/repository/RepositoryBranchesViewModel.kt

```diff
@@ -0,0 +1,72 @@
+package com.bottlerocketstudios.brarchitecture.ui.repository
+
+import com.bottlerocketstudios.brarchitecture.R
+import com.bottlerocketstudios.brarchitecture.data.model.BranchDto
+import com.bottlerocketstudios.brarchitecture.data.model.TargetDto
+import com.bottlerocketstudios.brarchitecture.domain.models.Branch
+import com.bottlerocketstudios.brarchitecture.domain.repositories.BitbucketRepository
+import com.bottlerocketstudios.brarchitecture.ui.BaseViewModel
+import com.bottlerocketstudios.compose.repository.RepositoryBranchItemUiModel
+import com.bottlerocketstudios.compose.util.formattedUpdateTime
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.map
+import org.koin.core.component.inject
+import timber.log.Timber
+import java.time.Clock
+
+class RepositoryBranchesViewModel : BaseViewModel() {
+    // DI
+    private val repo: BitbucketRepository by inject()
+    private val clock by inject<Clock>()
+
+    // State
+    private val srcBranches = MutableStateFlow<List<BranchDto>>(emptyList())
```

Comment:
Just had a quick chat with Luis about taking these DTO objects out of the VM

Reply:
Made the change, DTO is donzo